src/System.Windows.Forms/src/System/Windows/Forms/ListView.ListViewAccessibleObject.cs

```diff
@@ -51,7 +51,7 @@ private bool OwnerHasDefaultGroup
             }
 
             private bool OwnerHasGroups
-                => _owningListView.IsHandleCreated && _owningListView.Groups.Count > 0;
+                => _owningListView.IsHandleCreated && _owningListView.GroupsEnabled && _owningListView.Groups.Count > 0;
```

Comment:
```suggestion
                => _owningListView.IsHandleCreated && _owningListView.GroupsEnabled;
```
https://github.com/dotnet/winforms/blob/775aef04bf1f5d2c8af3ff01c622749438bc464f/src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs#L1007-L1013